pkg/virt-launcher/virtwrap/util/libvirt_helper.go

```diff
@@ -450,12 +450,6 @@ func configureQemuConf(qemuFilename string) (err error) {
 	}
 	defer util.CloseIOAndCheckErr(qemuConf, &err)
 
-	// We are in a container, don't try to stuff qemu inside special cgroups
-	_, err = qemuConf.WriteString("cgroup_controllers = [ ]\n")
```

Comment:
Now all builds need to include this. Can we make this into release note?

Reply:
If you think it brings value yes we can but I don't think we should worry or support custom builds, it's the opposite, they should track upstream and at least use the sensible default settings.

Reply:
Release note updated!